src/SamplesApp/UITests.Shared/Windows_UI_Xaml/Resources/XamlLocalResources.xaml

```diff
@@ -0,0 +1,19 @@
+﻿<Page
+	x:Class="UITests.Shared.Windows_UI_Xaml.Resources.XamlLocalResources"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:local="using:UITests.Shared.Windows_UI_Xaml.Resources"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	mc:Ignorable="d"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Page.Resources>
+		<local:CustomTemplateSelector x:Name="resourceWithAName" />
+		<SolidColorBrush x:Name="Blue2">Blue</SolidColorBrush>
+		<Style TargetType="TextBlock" x:Name="myStyle">
+		</Style>
+	</Page.Resources>
+
+	<!-- This is not a real sample, it's just to ensure the previous resources are not generating any compilation errors -->
```

Comment:
That's what `XamlGenerationTests` is for, could go there ;)

Reply:
You're right ;-)